src/bucket.js

```diff
@@ -71,6 +81,7 @@ export default class Bucket {
    * @param  {Object} options.headers The headers object option.
    * @return {Promise<Object, Error>}
    */
+  @nobatch("Batch doesn't support read operations.")
```

Comment:
Isn't it the same for all of them:

https://github.com/Kinto/kinto-client/pull/72/files#diff-d2dbe7779d59d4dd594e92248caba0a5R245

Why would you prevent people of doing a read operation in a batch if they plan on doing so?